code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -378,6 +378,8 @@
 /obj/effect/mob_spawn/human/demonic_friend/Initialize(mapload, datum/mind/owner_mind, obj/effect/proc_holder/spell/targeted/summon_friend/summoning_spell)
 	. = ..()
 	owner = owner_mind
+	if(!owner)
+		return
```

Comment:
Should this just return `INITIALIZE_HINT_QDEL` or is it possible for this to still be used by a ghost?

Reply:
it it gets initialized normally there shouldn't be any reason for this to return early so it should be fine trough i will put the return hint in